### DISCUSSION TITLE
I don't see my stats

### DISCUSSION THREAD

**Discussion by TanoTech** [community/community#113530] [Repositories]
### Select Topic Area

Bug

### Body

Hello everyone, 
I'm working in a group project and I did a a little bit of commits and important addition to the code,
but I do not see them in the repo stats, what can it be?
https://github.com/valkqt/Veterinario3/graphs/contributors?from=2024-03-10&to=2024-03-21&type=c

**Comment by SebastienXia** [comment]
## Hello, @TanoTech 👋

😌 Simply go to the activity category of your repo : https://github.com/valkqt/Veterinario3/activity

`[close the thread and mark as answered if this helped you resolve your issue]`
